### ISSUE TITLE
Gimbal breaking on configs

### ISSUE THREAD

**Issue by Capkirk123** [NathanKell/ModularFuelSystem#240]
If an engine has gimballing capability, but it has a config without gimbal (set by using 			gimbalRange = 0 in ModuleEngineConfigs), then selecting that config will permanently break the gimbal on that part, even if other configs are selected. The gimbal will only be restored if the part is replaced.

To reproduce:

1. Select and engine that has a gimbal on some configs, and none on others (the NK15/33 is a good example, if you have RO)

2. Observe the engine gimbal functions, either by using RCS build aid or launching the vessel

3. Select a config with 0 degrees of gimbal, and then select the original config

4. Observe the gimbal no longer functions, using RCS build aid or by launching the vessel

**Comment by Starwaster** [comment]
Can you look at the config for that engine and check if the other (non-zero) configs have the gimbalRange field at all?

I have a suspicion as to why this might happen but my PC is down and I’m limited to my iPhone. 

But what I think is happening is that some configs are lacking explicit gimbalRange definition. If such a config is loaded it should fallback to the base engine config. It sounds like it is not.

**Comment by Starwaster** [comment]
Also, Explicitly defining gimbal for all configs should resolve.

**Comment by Capkirk123** [comment]
Yes, explicitly defining the gimbal range for all configs fixes the problem.

**Comment by Starwaster** [comment]
If you’re in the RO/RP1 discord channel, please tell them I suggest explicit gimbal config for all MEC configs if any MEC touches the gimbal. 

My PC is down and I’m waiting on a replacement PSU but supply line disruption means I have no idea when it’ll get here. 

When I’m back up and running I’ll  make code changes so it has a fallback. (Thought I had already but maybe not. Don’t feel like code diving on my phone)
